include/dlaf/eigensolver/reduction_to_band/impl.h

```diff
@@ -1125,7 +1126,8 @@ common::internal::vector<pika::shared_future<common::internal::vector<T>>> Reduc
     // blocked waiting to do nothing on the next iteration (computePanel), while other ranks would be
     // stuck waiting for it for completing steps of the previous iteration, needed for the update of the
     // trailing matrix that will unlock the next iteration.
-    trigger_panel = pika::when_all(selectRead(x, x.iteratorLocal()), selectRead(xt, xt.iteratorLocal()));
+    trigger_panel = pika::future<void>(
```

Comment:
Just checking my understanding: could `drop_value` be used in place of `future<void>`?

Moreover in general I'm wondering if we could switch to something more "senderified", but probably we would need to use `selectRead` @teonnik variant (I remember of something he may have done in #436)

Reply:
> Just checking my understanding: could `drop_value` be used in place of `future<void>`?

Actually, yes, that's a good point. The `select*` functions are the blocker for full senderification, but in fact I think this could be further senderified already now without that. I'll push some changes for that.
 
> Moreover in general I'm wondering if we could switch to something more "senderified", but probably we would need to use `selectRead` @teonnik variant (I remember of something he may have done in #436)

Yes, it should be updated. This issue tracks that: https://github.com/eth-cscs/DLA-Future/issues/605. The main blocker is that some tests rely on checking if a future is ready or not. Please comment over there if you have thoughts or ideas.

Reply:
In theory `pika::execution::experimental::{when_all,when_all_vector,drop_value}` could've been used here, but because of the usual problem of `Tile`s not being copyable from `shared_future`s without `keep_future` I've reverted the blind change. This is better updated as part of #605.